examples/kafka/confluent-s3-connector/README.md

```diff
@@ -0,0 +1,166 @@
+```bash
+$ kubectl -n kafka run kafka-producer -ti --image=strimzi/kafka:0.20.0-kafka-2.6.0 --rm=true --restart=Never -- bin/kafka-console-producer.sh --broker-list my-cluster-kafka-bootstrap:9092 --topic blogpost
+
+>{"title":"The Matrix","year":1999,"cast":["Keanu Reeves","Laurence Fishburne","Carrie-Anne Moss","Hugo Weaving","Joe Pantoliano"],"genres":["Science Fiction"]}
+>{"title":"ABCD3","year":2000,"cast":["Keanu Reeves","Laurence Fishburne","Carrie-Anne Moss","Hugo Weaving","Joe Pantoliano"],"genres":["Science Fiction"]}
+>{"title":"Student of the year","year":2001,"cast":["Keanu Reeves","Laurence Fishburne","Carrie-Anne Moss","Hugo Weaving","Joe Pantoliano"],"genres":["Science Fiction"]}
+>{"title":"ABCD","year":2002,"cast":["Keanu Reeves","Laurence Fishburne","Carrie-Anne Moss","Hugo Weaving","Joe Pantoliano"],"genres":["Science Fiction"]}
+```
+* check S3 bucket for the topic
+
+## Perform Restore
+To perform restore, a prehook restore operation is performed which will purge all events from the topics in the kafka cluster whose backups were performed previously.
+```bash
+# Create a restoreprehook actionset
+$ kanctl create actionset --action restoreprehook --namespace kasten-io --blueprint kafka-blueprint --profile kafka/s3-profile-fn64h --objects v1/configmaps/kafka/s3config
+
+```
+**NOTE:**
+* Here the topic needs to be already present in the kafka cluster.
+* Before running prehook operation confirm that no other consumer is consuming data from that topic
+
+Perform the restore operation
+
+```bash
+# Perform a restore actionset
+$ kanctl create actionset --action restore --namespace kasten-io --blueprint kafka-blueprint --profile kafka/s3-profile-fn64h --objects v1/configmaps/kafka/s3config
+
+```
+## Verify restore
+Create a consumer for topics
+```bash
+# creating a consumer on a different terminal
+$ kubectl -n kafka run kafka-consumer -ti --image=strimzi/kafka:0.20.0-kafka-2.6.0 --rm=true --restart=Never -- bin/kafka-console-consumer.sh --bootstrap-server my-cluster-kafka-bootstrap:9092 --topic blogpost --from-beginning
+```
+All the messages restored can be viewed
+
+## Delete Blueprint and Profile CR
+
+```bash
+# delete the blueprint
+$ kubectl delete blueprints.cr.kanister.io <blueprint-name> -n kasten-io
+# Get the profile
+$ kubectl get profiles.cr.kanister.io -n kafka
+NAME               AGE
+s3-profile-fn64h   2h
+# Delete the profile
+$ kubectl delete profiles.cr.kanister.io s3-profile-fn64h -n kafka
+```
+
+### Troubleshooting
+
+If you run into any issues with the above commands,
+
+you can check the logs of the controller using:
+```bash
+$ kubectl --namespace kasten-io logs -l run=kanister-svc -f
+```
+you can check events of the actionset:
+```bash
+$ kubectl describe actionset <actionset-name> -n kasten-io
+```
+you can also check the logs of kanister job
+```bash
+# get the pod name
+$ kubectl get pod -n kafka
+
+# check the logs
+$ kubectl logs <name-of-pod-running the job> -n kafka
```

Comment:
```suggestion
Check events of the ActionSet:
```bash
$ kubectl describe actionset <actionset-name> -n kasten-io
```
Check the logs of Kanister job
```bash
# Get the Kanister job pod name
$ kubectl get pod -n kafka

# Check the logs
$ kubectl logs <name-of-pod-running the job> -n kafka
```